Lib/sqlite3/__main__.py

```diff
@@ -70,8 +70,9 @@ def runsource(self, source, filename="<input>", symbol="single"):
                     pass
                 case _ as unknown:
                     t = theme.traceback
-                    self.write(f'{t.type}Error{t.reset}:{t.message} unknown'
-                               f'command or invalid arguments:  "{unknown}".\n{t.reset}')
+                    print(f'{t.type}Error{t.reset}:{t.message} unknown command '
```

Comment:
Note that the banner and tracebacks are written using `self.write()`. Why not use it here for consistence?

Reply:
Errors (35) use print? And so do the other match statements branches.

Reply:
Other branches output to stdout.

Line 35 perhaps should use `self.write()` too.

Reply:
We can leave `execute()` for now. It is not worth to rewrite it just for this.

Reply:
```suggestion
                    self.write(f'{t.type}Error{t.reset}:{t.message} unknown command '
```
It is up to you/Erlend. The box to allow you to commit was checked.